sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -340,10 +341,6 @@ class CG {
 
   CG(CG &&CommandGroup) = default;
 
-  std::vector<Requirement *> getRequirements() const { return MRequirements; }
-
-  std::vector<detail::EventImplPtr> getEvents() const { return MEvents; }
-
```

Comment:
Why delete this functions instead of having these return const references?

Reply:
Because I need non const reference. I decided that having direct access to the member is more clear then having getter which just returns non-const reference.